### ISSUE TITLE
jakarta compatibility on generated class `AbstractGroupsManager`

### ISSUE THREAD

**Issue by ykevinc** [indeedeng/proctor#163]
Hi,

Thanks for open sourcing this AB testing library. 

With
```
import jakarta.servlet.http.HttpServletRequest;
import jakarta.servlet.http.HttpServletResponse;
```
superseded
```
import javax.servlet.http.HttpServletRequest;
import javax.servlet.http.HttpServletResponse;
```

https://github.com/spring-projects/spring-boot/wiki/Spring-Boot-3.0.0-M1-Release-Notes
https://stackoverflow.com/questions/71179660/spring-boot-3-jakarta-and-javax

Will there be any plan on switching the import over to jakarta?

Thank you.

**Comment by SwapnilUgalmugale** [comment]
Yes you should import 'jakarta' over to 'javax'.While your Spring Boot application may currently function properly with both 'javax' and 'jakarta' libraries due to Spring Boot's backward compatibility, it's crucial for future compatibility with Jakarta EE 9 and Jakarta-compatible servers like Tomcat 10 its is recommended by spring boot 3 release notes to use imports from 'javax' to 'jakarta'.
